scripts/controllers/CartesianPositionController.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python
+
+from PandaController import PandaController
+import numpy as np
+import rospy
+import tf
+
+from ros_robotic_skin.srv import getJacobian
+from sensor_msgs.msg import JointState
+
+
+Q_MIN = np.array([-2.8973, -1.7628, -2.8973, -3.0718, -2.8973, -0.0175, -2.8973])
+Q_MAX = np.array([+2.8973, +1.7628, +2.8973, -0.0698, +2.8973, +3.7525, +2.8973])
```

Comment:
Are these values always the same? I think it would be better to add them to the parameters file, then read them in.